docs/intro/tutorial01.txt

```diff
@@ -79,14 +87,12 @@ Let's look at what :djadmin:`startproject` created:
 
 These files are:
 
-* The outer :file:`mysite/` root directory is a container for your project. Its
-  name doesn't matter to Django; you can rename it to anything you like.
-
+* The outer :file:`djangotutorial/` root directory is a container for your project.
```

Comment:
I'd revert the "Its name doesn't matter..." as per comment above.

Reply:
Thank you for reading my changes again. :)